apps/openmw/mwrender/groundcover.cpp

```diff
@@ -110,32 +99,14 @@ namespace MWRender
 
             // Display lists do not support instancing in OSG 3.4
             geom.setUseDisplayList(false);
+            geom.setUseVertexBufferObjects(true);
```

Comment:
So we're going full VBO here and avoiding DL?

Reply:
In practical terms, OSG pretty much always would have used VBOs before this was added because that's the default on VBO-compatible hardware with disabled display lists. It's probably better to be explicit, though.